templates/containers/docker-ubuntu.html

```diff
@@ -21,7 +21,7 @@
       <h1>Docker Engine on Ubuntu</h1>
       <h2>Integrated support service with enterprise&nbsp;SLAs</h2>
       <p>Docker Engine is a lightweight container runtime with robust tooling that builds and runs your container. Over 65% of all Docker-based scale out operations run on Ubuntu.</p>
-      <p>Canonical works closely with Docker Inc. to deliver an integrated Commercially Supported (CS) Docker Engine offering on Ubuntu. This partnership provides customers with the fastest and easiest path for support of Ubuntu and Docker Engine in enterprise Docker operations.</p>
+      <p>Canonical works closely with Docker Inc. to deliver an integrated Enterprise Edition Docker offering on Ubuntu. This partnership provides customers with the fastest and easiest path for support of Ubuntu and Docker Engine in enterprise Docker operations.</p>
```

Comment:
Enterprise Edition Docker here and Docker Enterprise Edition below?  if it is a product name, it should be the same everywhere, no?